gradle/version-info.gradle

```diff
@@ -0,0 +1,78 @@
+import java.security.MessageDigest
```

Comment:
can you add some documentation what this gradle file does?

Reply:
Will add.